ros2nodl/ros2nodl/_verb/_show.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2020 Canonical Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License Version 3 as published by
+# the Free Software Foundation.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import pprint
+import shutil
+import sys
+
+import nodl
+from ros2cli.verb import VerbExtension
+from ros2pkg.api import package_name_completer, PackageNotFoundError
+from ros2run.api import ExecutableNameCompleter
+
+
+class _ShowVerb(VerbExtension):
+    """Show NoDL data."""
+
+    def add_arguments(self, parser: argparse.ArgumentParser, cli_name: None = None):
+        # Ignoring type because of https://github.com/python/typeshed/issues/1878
+        parser.add_argument(  # type: ignore
+            'package_name', help='Name of the package to show.'
+        ).completer = package_name_completer
+
+        # Ignoring type because of https://github.com/python/typeshed/issues/1878
+        parser.add_argument(  # type: ignore
+            'executables', nargs='*', default=[], help='Specific Executable to display.'
```

Comment:
Check your --help output, I think this will show up as `[executables [executables...]]` or similar. If so you might consider specifying the `metavar` arg here.

Reply:
fixed in 0c6cd25

Reply:
Hmm, was that commit lost?

Reply:
Ah  I subbed it in validate not here